spark/src/test/scala/geotrellis/spark/io/LayerUpdateSpaceTimeTileFeatureTests.scala

```diff
@@ -0,0 +1,113 @@
+package geotrellis.spark.io
+
+import geotrellis.proj4.LatLng
+import geotrellis.raster._
+import geotrellis.raster.testkit._
+import geotrellis.spark._
+import geotrellis.spark.io.avro.codecs._
+import geotrellis.spark.io.json._
+import geotrellis.spark.testkit._
+import geotrellis.spark.tiling._
+import geotrellis.spark.util._
+import geotrellis.util._
+import geotrellis.vector._
+
+import org.apache.spark.rdd._
+import org.joda.time.DateTime
+
+trait LayerUpdateSpaceTimeTileFeatureTests
```

Comment:
FunSpec, should be named `Spec` instead of `Tests`. `Tests` represents `FunSuite` tests.